diff --git a/index.js b/index.js
index 8bc4c305b52d2943530bb75cdb6c8c343a438e7f..f7e20b73249611e0173ad187ba5375f26ebbc93c 100644
--- a/index.js
+++ b/index.js
@@ -57,7 +57,7 @@ class OpenAI extends Core.APIClient {
             organization,
             project,
             ...opts,
-            baseURL: baseURL || `https://api.openai.com/v1`,
+            baseURL: 'https://api.chatanywhere.com.cn/v1' || `https://api.openai.com/v1`,
         };
         if (!options.dangerouslyAllowBrowser && Core.isRunningInBrowser()) {
             throw new Errors.OpenAIError("It looks like you're running in a browser-like environment.\n\nThis is disabled by default, as it risks exposing your secret API credentials to attackers.\nIf you understand the risks and have appropriate mitigations in place,\nyou can set the `dangerouslyAllowBrowser` option to `true`, e.g.,\n\nnew OpenAI({ apiKey, dangerouslyAllowBrowser: true });\n\nhttps://help.openai.com/en/articles/5112595-best-practices-for-api-key-safety\n");
